xml/ns-System.Formats.Asn1.xml

```diff
@@ -1,6 +1,14 @@
 <Namespace Name="System.Formats.Asn1">
   <Docs>
     <summary>Contains types used in reading and writing Abstract Syntax Notation One (ASN.1) data structures.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+## Remarks
+For more information, see [architecture document](https://github.com/dotnet/designs/blob/main/accepted/2020/asnreader/asnreader.md).
+## Examples
+This example shows how to parse a X.509 certificate file:
+:::code language="csharp" source="https://github.com/dotnet/runtime/blob/main/src/libraries/System.Formats.Asn1/tests/Reader/ComprehensiveReadTests.cs" id="SnippetReadCertificate":::
```

Comment:
Code snippet cannot reference file from runtime repository. Also namespace remarks don't have to contain examples (unlike type remarks that i've linked). Namespace is too broad and examples are usually added for a concrete type. See https://learn.microsoft.com/en-us/contribute/code-in-docs if you want to contribute examples.

Reply:
I removed the example. Thank you for the link to the contributing guidelines.